### DISCUSSION TITLE
How to define an early stop?

### DISCUSSION THREAD

**Discussion by lcoandrade** [azavea/raster-vision#1812] [Q&A] [answered]
How can I define an early stop with RV?

**Comment by AdeelH** [comment, marked as answer]
You can't currently (#1720). You may instead try one of the following options:
- subclass whichever `Learner` class you're using and override its training-related methods
- use [PyTorch Lightning](https://docs.rastervision.io/en/0.20/usage/tutorials/lightning_workflow.html) for training
- write your own custom training loop
